### DISCUSSION TITLE
Unit price and sale order

### DISCUSSION THREAD

**Discussion by irameez** [inventree/InvenTree#2894] [Q&A] [answered]
I am trying to learn how the pricing works in inventree, I use it to keep track of item we manufacture. So I create a new part and then enter the stock we have.
I have a question about unit price, after I define a price break in pricing tab shouldn't it automatically pick up that price when I enter the part in sale order. Currently it asks for the price to be manually added in the sale order.

Thanks,
Rameez,

**Comment by matmair** [comment, marked as answer]
There is a button with a dollar sign on each line in the purchase order tab. If you press that it shows you the calculation and you can choose to update the price.

**Reply by irameez** [reply]
But we don't purchase the items we manufacture them.

**Reply by matmair** [reply]
The function is also available for sales orders - also with the dollar sign as a button per line. I did not read your question with enough attention, I am sorry about that.

**Reply by irameez** [reply]
yes, I can see that in the sales order, but my question was should it not pick up the price automatically from the price tab? it asks me to enter it manually in sales order too.

**Reply by matmair** [reply]
@irameez this sounds more like a feature request. Please open an issue and fill out the [template](https://github.com/inventree/InvenTree/issues/new?assignees=&labels=enhancement&template=feature_request.md&title=%5BFR%5Dpricecalculation).

**Reply by SchrodingersGat** [reply]
@irameez the pricing features in InvenTree are reasonably underdeveloped compared to other areas. Please open an issue as @matmair suggests. We would also appreciate development assistance if you are able to help!
